q2_gneiss/composition/_composition.py

```diff
@@ -56,3 +56,29 @@
                                       'ilr transform.')},
     description="Calculate balances given a hierarchy."
 )
+
+
+plugin.methods.register_function(
+    function=ilr_phylogenetic,
+    inputs={'table': FeatureTable[Composition],
+            'tree': Phylogeny[Rooted]},
+    outputs=[('balances', FeatureTable[Balance]),
+             ('tree', Phylogeny[Rooted])],
```

Comment:
Can we convert this tree to a hierarchy so that the regression summaries don't crash downstream?